src/features/cartographie/components/lieux-mediation-numerique-details/webinaire/webinaire.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="d-flex flex-column">
+  <small class="d-flex text-muted-dark">
+    <i class="ri-timer-fill me-1" aria-hidden="true"></i>
+    45 min - 1 fois par mois
+  </small>
+  <div class="d-flex align-items-center my-5 flex-column flex-grow-1">
+    <h5 class="h5 text-primary fw-bold">Webinaire</h5>
```

Comment:
```suggestion
    <h? class="h5 text-primary fw-bold">Webinaire</h?>
```
Je ne pense pas qu'il faut mettre un h5 ici, pour l'accessibilité, il est important que la hiérarchie des titres soit respectée, il ne doit pas y avoir de hx manquant par exemple tu ne peut pas mettre un h3 après un h1 s'il n'y a pas de h2 entre les deux.
Si c'est le style du h5 que tu veux, tu peux utiliser la classe h5
![image](https://github.com/anct-cartographie-nationale/client-application/assets/68475403/c1573525-5a1a-4944-bcbc-b3ed66171889)